enterprise/frontend/src/embedding-sdk/components/public/InteractiveDashboard/EditableDashboardView.tsx

```diff
@@ -0,0 +1,124 @@
+import type { Query } from "history";
+import type React from "react";
+import { connect, type ConnectedProps } from "react-redux";
+import _ from "underscore";
+
+import type { SdkPluginsConfig } from "embedding-sdk";
+import * as dashboardActions from "metabase/dashboard/actions";
+import type { NavigateToNewCardFromDashboardOpts } from "metabase/dashboard/components/DashCard/types";
+import { Dashboard } from "metabase/dashboard/components/Dashboard/Dashboard";
+import {
+  getClickBehaviorSidebarDashcard,
+  getDashboardBeforeEditing,
+  getDashboardComplete,
+  getDocumentTitle,
+  getFavicon,
+  getIsAdditionalInfoVisible,
+  getIsAddParameterPopoverOpen,
+  getIsDashCardsLoadingComplete,
+  getIsDashCardsRunning,
+  getIsDirty,
+  getIsEditing,
+  getIsEditingParameter,
+  getIsHeaderVisible,
+  getIsNavigatingBackToDashboard,
+  getIsSharing,
+  getLoadingStartTime,
+  getParameterValues,
+  getSelectedTabId,
+  getSidebar,
+  getSlowCards,
+} from "metabase/dashboard/selectors";
+import type {
+  DashboardFullscreenControls,
+  DashboardRefreshPeriodControls,
+} from "metabase/dashboard/types";
+import type { PublicOrEmbeddedDashboardEventHandlersProps } from "metabase/public/containers/PublicOrEmbeddedDashboard/types";
+import { useDashboardLoadHandlers } from "metabase/public/containers/PublicOrEmbeddedDashboard/use-dashboard-load-handlers";
+import { closeNavbar, setErrorPage } from "metabase/redux/app";
+import { getIsNavbarOpen } from "metabase/selectors/app";
+import {
+  canManageSubscriptions,
+  getUserIsAdmin,
+} from "metabase/selectors/user";
+import type { DashboardId } from "metabase-types/api";
+import type { State } from "metabase-types/store";
+
+const mapStateToProps = (state: State) => {
```

Comment:
I'm a bit taken aback since the file name is `*View`, but this component sure connects both props and actions, which is the exact definition of a container (non-view) component.

The component is fine in my opinion, but it shouldn't be named as `*View`